tests/mypy/modules/fail4.py

```diff
@@ -24,3 +24,13 @@ def bar() -> str:
 
 # return type should be a string
 y: int = bar()
+
+
+# nested decorator should not produce an error
+@validate_arguments(config={'arbitrary_types_allowed': True})
```

Comment:
Why not put it in `success.py` instead of `fail4.py`?

Reply:
done